libs/application/core/src/types/Form.ts

```diff
@@ -89,6 +89,8 @@ export interface ExternalDataProvider extends FormItem {
   readonly children: undefined
   isPartOfRepeater?: boolean
   dataProviders: DataProviderItem[]
+  checkboxLabel?: MessageDescriptor | string
+  subTitle?: MessageDescriptor | string
```

Comment:
Should this type be like this in the future or just a `MessageDescriptor`  or just a `string` ?

Reply:
This is how things are currently typed in the application system. What I would like to see is just `string` and if there needs to be some processing of the value it should be done before passing it down as a prop.